iocore/net/UnixUDPNet.cc

```diff
@@ -1014,8 +1014,8 @@ bool
 UDPNetProcessor::CreateUDPSocket(int *resfd, sockaddr const *remote_addr, Action **status, NetVCOptions const &opt)
 {
   int res = 0, fd = -1;
-  int local_addr_len;
   IpEndpoint local_addr;
+  int local_addr_len = sizeof(local_addr.sa);
```

Comment:
Am I reading this right that this is only used as a throw-away, non-read output variable below? If so, I suggest just declaring and initializing it to zero just before the point of use.

Reply:
I thought so too, but actually initializing it with 0 is wrong.

> getsockname() returns the current address to which the socket
       sockfd is bound, in the buffer pointed to by addr.  The addrlen
       argument should be initialized to indicate the amount of space
       (in bytes) pointed to by addr.  On return it contains the actual
       size of the socket address.

https://man7.org/linux/man-pages/man2/getsockname.2.html

Reply:
Ah, good to know. Thanks.